configurator/DOCS.md

```diff
@@ -71,6 +71,6 @@ In case you've found a bug, please [open an issue on our GitHub][issue].
 
 [discord]: https://discord.gg/c5DvZ4e
 [forum]: https://community.home-assistant.io
-[issue]: https://github.com/home-assistant/hassio-addons/issues
+[issue]: https://github.com/home-assistant/addons/issues
 [reddit]: https://reddit.com/r/homeassistant
-[repository]: https://github.com/addons/repository
+[repository]: https://github.com/hassio-addons/repository
```

Comment:
That seems wrong

Reply:
@agners Why?

Reply:
This fixes an issue with the larger PR #2919.

Reply:
Hm I see. But that URL is not really used here, we don't point to the community add-ons.

Just drop it then.

```suggestion
```